docs/install-new.md

```diff
@@ -46,9 +46,13 @@ The steps below require the following software to be installed on your workstati
 
 ## Setting up Application Gateway Ingress Controller on AKS
 
-Once, we have the created the resources in Azure, we need to deploy following controllers on the cluster:
-1) `aad pod identity` - This controller uses user assigned identity and provide ARM access token to the controller.
-2) `application gateway ingress controller` - This controller communicates ingress related events to the Application Gateway resource.
+### Overview
+With the instructions in the previous section we created and configured a new Azure Kubernetes Service (AKS) cluster. We are now ready to deploy new Kubernetes infrastructure. The instructions below will guide us through the proccess of installing the following 2 components on our new AKS:
```

Comment:
is it suppose to be 'ready to deploy `to our` new Kubernetes infrastructure' ?

Reply:
I like this clarification - thanks!